SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -360,6 +361,12 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	// Clears all the snapshots taken, not meant to be used directly.
 	static void ClearAllTakenSnapshots();
 
+	// Get the player controller owned by the current flow.
+	APlayerController* GetFlowPlayerController();
+
+	// Get the pawn that belongs to the PlayerController owned by the current flow.
+	ATestMovementCharacter* GetFlowPawn();
```

Comment:
I would do just APawn* here, will also remove the need for the forward declare.
The tests can cast it themselves, if they expect something specific.